docs/book/themes/bootstrap-theme.rst

```diff
@@ -69,7 +69,7 @@ Install missing BootstrapTheme dependencies
 .. code-block:: bash
 
     yarn add sass-loader@^7.0.0 node-sass lodash.throttle -D
-    yarn add bootstrap bootstrap.native glightbox axios form-serialize @fortawesome/fontawesome-svg-core @fortawesome/free-brands-svg-icons @fortawesome/free-regular-svg-icons @fortawesome/free-solid-svg-icons
+    yarn add bootstrap@^4.5.0 bootstrap.native@^3.0.0 glightbox axios form-serialize @fortawesome/fontawesome-svg-core @fortawesome/free-brands-svg-icons @fortawesome/free-regular-svg-icons @fortawesome/free-solid-svg-icons
```

Comment:
I compared this line to the one from BootstrapTheme's repository, and we miss `popper.js` dependency here. Would you like to add it? Remember to do steps from the first comment generated after switch base :).